src/main/java/org/openhds/errors/model/ErrorLog.java

```diff
@@ -1,9 +1,11 @@
 package org.openhds.errors.model;
 
+import org.hibernate.validator.constraints.NotEmpty;
```

Comment:
I think we should not use hibernate annotations if we can help it.  How about javax size with min=1?
https://docs.oracle.com/javaee/7/api/javax/validation/constraints/Size.html